cluster/gce/upgrade-aliases.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# !!!EXPERIMENTAL!!! Upgrade a k8s cluster from routes to IP aliases for
+# node connectivity on GCE. This is only for migration.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ "${KUBERNETES_PROVIDER:-gce}" != "gce" ]]; then
+  echo "!!! ${1} only works on GCE" >&2
+  exit 1
+fi
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${KUBE_ROOT}/hack/lib/util.sh"
+source "${KUBE_ROOT}/cluster/kube-util.sh"
+
+# Print the number of routes used for k8s cluster node connectivity.
+#
+# Assumed vars:
+#   PROJECT
+function get-k8s-node-routes-count() {
+  local k8s_node_routes_count=$(gcloud alpha compute routes list \
```

Comment:
Do we need to use the alpha API?

Reply:
Not for getting route. Removed alpha from gcloud cmd.